Document the new data maps (#50)
neoforged/Documentation: docs/datamaps/neo_maps.md

        }
    }
}
```

## `neoforge:furnace_fuels`
NeoForge provides a data map that allows configuring item burn times.  
This data map is located at `neoforge/data_maps/item/furnace_fuels.json` and its objects have the following structure:
```js
{
    // A positive integer representing the item's burn time, in ticks
    "burn_time": 1000
}
```

Example:
```js
{
    "values": {
        // Give anvils a 2 seconds burn time
        "minecraft:anvil": {
            "burn_time": 40
        }
    }
}
```

:::note
Other in-code methods like `IItemExtension#getBurnTime` will take priority over the data map, so it is recommended that you use the data map for simple, static burn times even in your mod so that users can configure them.
:::

:::warning

> Vanilla adds a burn time to logs and planks in the `minecraft:logs` and `minecraft:planks` tag. However, that tag also contains Nether wood, so a removal for elements in `#minecraft:non_flammable_wood` is added.  

Reviewer: ```suggestion
Vanilla adds a burn time to logs and planks in the `minecraft:logs` and `minecraft:planks` tag. However, those tags also contain Nether wood, so a removal for elements in `#minecraft:non_flammable_wood` is added.  
```

However, the removal does not affect any values added by other packs or mods, so if you want to change the values for the wood tags you will need to add a removal for the non flammable tag yourself.
:::

## `neoforge:parrot_imitations`

> NeoForge provides a data map that allows configuring the sounds produced by parrot when they want to imitate a mob, as a replacement for `Parrot#MOB_SOUND_MAP` (which is now ignored).  

Reviewer: ```suggestion
NeoForge provides a data map that allows configuring the sounds produced by parrots when they want to imitate a mob, as a replacement for `Parrot#MOB_SOUND_MAP` (which is now ignored).  
```

This data map is located at `neoforge/data_maps/entity_type/parrot_imitations.json` and its objects have the following structure:
```js
{
    // The ID of the sound that parrots will produce when imitating the mob
    "sound": "minecraft:entity.parrot.imitate.creeper"
}
```

Example:
```js
{
    "values": {
        // Make parrots produce the ambient cave sound when imitating allays
        "minecraft:allay": {
            "sound": "minecraft:ambient.cave"
        }
    }
}
```

## `neoforge:vibration_frequencies`

> NeoForge provides a data map that allows configuring the shulker vibration frequencies emmited by game events, as a replacement for `VibrationSystem#VIBRATION_FREQUENCY_FOR_EVENT` (which is now ignored).  

Reviewer: ```suggestion
NeoForge provides a data map that allows configuring the shulker vibration frequencies emitted by game events, as a replacement for `VibrationSystem#VIBRATION_FREQUENCY_FOR_EVENT` (which is now ignored).  
```
